app/tasks/raster_tile_set_assets/raster_tile_set_assets.py

```diff
@@ -67,13 +68,33 @@ async def raster_tile_set_asset(
 
     creation_options = PixETLCreationOptions(**co)
 
+    jobs: List[Job] = list()
     callback: Callback = callback_constructor(asset_id)
 
-    create_raster_tile_set_job: Job = await create_pixetl_job(
-        dataset, version, creation_options, "create_raster_tile_set", callback
+    if creation_options.unify_projection:
+        target_crs = "epsg:4326"
+        new_src_uris = list()
+        for i,_ in enumerate(creation_options.source_uri):
+            new_src_uris.append(
+                f"s3://{DATA_LAKE_BUCKET}/{dataset}/{version}/raster/{target_crs}/reprojected/SRC_{i}"
+            )
+        target_prefix = new_src_uris[0].rsplit("/", 1)[0]
+        jobs.append(
+            await create_unify_projection_job(
+                dataset, creation_options.source_uri, target_prefix, target_crs, "unify_projection", callback
+            )
+        )
+
+        creation_options.source_uri = new_src_uris
+
+
+    jobs.append(
+        await create_pixetl_job(
+            dataset, version, creation_options, "create_raster_tile_set", callback
```

Comment:
***In the case the `unify_projection` is True, don't you need to specify a parent dependency argument here?  The `create_unify_projection_job` must run fully before the `create_pixetl_job` runs.

Reply:
Yep, thanks!